packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -365,4 +365,64 @@ StructuredListMedia.args = {
       selected: false
     },
   ]
+}
+
+
+/*<------------------------------------ Select ---------------------------------------->*/
+
+const StructuredListSelectableTemplate = ({ listItems, striped, showHeader, headerId, context, sx }) => {
+  return ` 
+       <cbp-structured-list
+          ${striped ? `striped` : ''}
+          selectable
+          header-id="list-header"
+          ${context && context != 'light-inverts' ? `context=${context}` : ''}   
+          ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+        >
+        ${showHeader ? `<div slot="cbp-structured-list-header" id="${headerId}"><cbp-checkbox value='1'> <input type='checkbox' name='checkbox' value='1' /></cbp-checkbox> <span>XX results - Y filters Applied Updated: XX/XX/XXXX XX:XX:XX EST</span></div>` : ''}
```

Comment:
your checkbox doesn't have a label. It also doesn't need a name for the select all - you wouldn't want it submitted in a post. I don't think it needs a value either technically or what it should be.

Reply:
removed the name & value here, added 'hidden' label in same fashion as the generateSelectableItems is applying them to the list items checkbox